bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java

```diff
@@ -95,16 +95,27 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     case CHANNEL_LOADPOINT_MODE:
                         if (command instanceof StringType) {
                             evccAPI.setMode(loadpoint, command.toString());
+                        } else {
+                            throw new EvccException("Command has wrong type, StringType required!");
```

Comment:
Out of curiosity: Why throw exception here, which is caught locally within the same method, instead of controlling the flow by logging and returning? Throwing and handling exceptions is expensive.

Reply:
I first thought of logging, but then I thought, that exceptions are handled anyway, so let‘s throw an exception instead.
As you obviously prefer logging, I‘ll change that to logging,

Reply:
I've update the PR.